src/my-command.js

```diff
@@ -1,3 +1,65 @@
+const sketch = require('sketch')
+const UI = require('sketch/ui')
+
+function calcShadowY(elevation) {
+  if(elevation === 0) {
+    return '0'
+  }
+  return elevation / 2
```

Comment:
How about always returning string type because It's difficult to expect what type will be returned?